doc/CODING.md

```diff
@@ -122,9 +120,6 @@ Files should start with:
 	 *
 	 * @copyright BSD License (see doc/COPYING or http://www.libelektra.org)
 	 */
-	
-
-\endverbatim
```

Comment:
You are right – Thanks. I just deleted the responsible commit. I still think it is kind of strange to see Doxygen tags in a Markdown file.